python/tests/test_neutron.py

```diff
@@ -54,6 +54,38 @@ def test_neutron_convert_out_arg():
     assert dspacing is d
 
 
+def test_neutron_convert_with_calibration():
```

Comment:
Move this test into `diffraction/` subfolder?

Reply:
I'd say no since the export is not in a `diffraction` directory.

Reply:
Given that this is Python code, wouldn't it make more sense to follow the Python module structure, rather than the place of some C++ code?

Reply:
Since this is a developer focused thing I think it makes more sense to match file structure (to keep the test and the thing that is tested in similar locations).

Reply:
Or maybe the exports should be made to match the intended module structure.

Reply:
Or maybe the exports should be made to match the intended module structure.

Reply:
Restructuring the exports may need more consideration as the `python` directory is already a mix of benchmarks, tests, Python and C++. For now I've just made the Python tests match the module structure.